### ISSUE TITLE
Audio system consistently breaks when switching to a 44.1khz FLAC audio track

### ISSUE THREAD

**Issue by CrashOveride95** [mpvnet-player/mpv.net#573]
**Describe the bug**
Upon switching to a a 44.1khz audio track, the WASAPI audio system will crash resulting in video not being able to play back until a restart. Playing a 44.1khz FLAC by itself will not crash, however if you "switch" the audio track by clicking the track button, it will crash the same way. This bug is not reproducible in mainline mpv. Additionally, updating `libmpv-2.dll` gives the same results.

**To Reproduce**
1. Open a MKV with a 44.1khz FLAC track muxed in, or a 44.1khz FLAC file.
2. Switch the current audio track to the 44.1khz FLAC, either by clicking the audio button in the player, or in the right click menu.
3. WASAPI will crash and audio will fail to play.
4. in the command line log, if open, this error will be printed out:
```
[ao/wasapi] Error activating device: AUDCLNT_E_DEVICE_INVALIDATED (0x88890004)
[ao/openal] Direct channels aren't supported by this version of OpenAL
AO: [openal] 44100Hz stereo 2ch s16
AV: 00:00:03 / 00:24:17 (0%) A-V:  0.000
Audio device underrun detected.

Audio/Video desynchronisation detected! Possible reasons include too slow
hardware, temporary CPU spikes, broken drivers, and broken files. Audio
position will not match to the video (see A-V status field).
```

**Expected behavior**
Video and audio playback to continue as normal

**Screenshots**
![mpvbug](https://github.com/mpvnet-player/mpv.net/assets/40003130/4a03fdb5-20c6-469c-8af2-bbc319fa4445)

**Additional context**
1.  ![mpvver](https://github.com/mpvnet-player/mpv.net/assets/40003130/d6a16588-1c1e-4081-a60d-0aa1486c4784)
2. Windows 10 Version 22H2
5. Nvidia GeForce GTX 1660 Ti
6. [Media info](https://hastebin.skyra.pw/ezoraxatit.yaml)

**Comment by zhongfly** [comment]
Can't reproduce with mpv v0.36.0-712-gb6703873 (2023/11/2)
Try latest mpv ( https://github.com/zhongfly/mpv-winbuild/releases/tag/2023-11-02-b670387 )

sample file same with you :`[Crash] Lupin III Part 1 Remux (+Extras) | Lupin the Third | Lupin the 3rd | ルパン三世` Lupin III Part 1 - EP16 [461E3EBC].mkv 

[cut.zip](https://github.com/mpvnet-player/mpv.net/files/13240723/cut.zip)

**Comment by CrashOveride95** [comment]
Yes, as I stated in the issue comment, I tested with mainline mpv and experienced no issues, but got a crash on mpv.net
I'm not sure what could cause this to happen

**Comment by zhongfly** [comment]
> Yes, as I stated in the issue comment, I tested with mainline mpv and experienced no issues, but got a crash on mpv.net I'm not sure what could cause this to happen

No, What I said earlier is mpvnet and cannot be reproduced on mpv.net

**Comment by CrashOveride95** [comment]
In your initial edit you stated mpv; I can see now you fixed it to libmpv, apologies for confusion
I tried the newest libmpv build and still get the same error, yet the `mpv` build from the same release works just fine
I really don't know why this isn't reproducible :(

**Comment by zhongfly** [comment]
Based on the error, I would venture a guess that it might have something to do with your audio playback device. Maybe switch to another audio playback device ?
Maybe you can provide a more detailed log,`mpvnet xxx.mkv --no-config --log-file=log.txt`.
Then drag and drop that log.txt file into the comment box to upload the attachment. Never copy and paste the contents of the log directly, that's just too much!

**Comment by CrashOveride95** [comment]
Here's the log : 
[log.txt](https://github.com/mpvnet-player/mpv.net/files/13255607/log.txt)

**Comment by CrashOveride95** [comment]
Updating to the latest stable of mpv.net (v7.1.0.0) fixed the issue, and 44.1khz audio is working as expected
Not sure why it wasn't working before, but it's all good now so I will close this issue
